bitnami/memcached/Chart.yaml

```diff
@@ -22,4 +22,4 @@ name: memcached
 sources:
   - https://github.com/bitnami/bitnami-docker-memcached
   - http://memcached.org/
-version: 5.10.2
+version: 5.10.3
```

Comment:
Could you bump the minor version (i.e. 5.10.x -> 5.11.0)? Since it is adding new features to the chart.

Reply:
Sorry, my mistake. I already fix it